src/Layout/redist/targets/BundledSdks.targets

```diff
@@ -12,4 +13,38 @@
     <BundledSdk Include="Microsoft.SourceLink.GitLab" Version="$(MicrosoftSourceLinkGitLabVersion)" />
     <BundledSdk Include="Microsoft.SourceLink.Bitbucket.Git" Version="$(MicrosoftSourceLinkBitBucketGitVersion)" />
   </ItemGroup>
+
+  <!-- Restore bundled sdks -->
+  <ItemGroup>
+    <PackageDownload Include="@(BundledSdk)" Version="[%(Version)]" />
+  </ItemGroup>
+
+  <Target Name="PublishSdks">
+    <ItemGroup>
+      <BundledSdk SdkLayoutDirectory="$(OutputPath)Sdks\%(Identity)\"
+                  PackageNameLowercase="$([System.String]::new('%(Identity)').ToLowerInvariant())" />
+      <BundledSdk NuPkgPath="$(NuGetPackageRoot)%(PackageNameLowercase)\%(Version)\" />
+    </ItemGroup>
+
+    <Error Text="Bundled SDK has not been restored yet: '%(BundledSdk.NuPkgPath)%(BundledSdk.PackageNameLowercase).nuspec'"
+           Condition="!Exists('%(BundledSdk.NuPkgPath)%(BundledSdk.PackageNameLowercase).nuspec')" />
+
+    <ItemGroup>
+      <SdkContent Include="%(BundledSdk.NuPkgPath)**\*"
+                  Exclude="%(BundledSdk.NuPkgPath)Icon.png;
+                           %(BundledSdk.NuPkgPath)%(BundledSdk.PackageNameLowercase).nuspec;
+                           %(BundledSdk.NuPkgPath)%(BundledSdk.PackageNameLowercase).%(BundledSdk.Version).nupkg;
+                           %(BundledSdk.NuPkgPath)%(BundledSdk.PackageNameLowercase).%(BundledSdk.Version).nupkg.sha512;
+                           %(BundledSdk.NuPkgPath)**\*.nupkg.metadata;
+                           %(BundledSdk.NuPkgPath)**\*.signature.p7s"
```

Comment:
aren't these two in the root folder as well? why `**`?

Reply:
Yes, but not sure if they could exist in subfolders as well.

Reply:
they don't